config/Dockerfile

```
@@ -0,0 +1,21 @@
+FROM ubuntu:latest
+RUN apt-get -y update && apt-get -y clean 
+RUN apt-get -y install sudo
+RUN apt-get -y install git
+RUN sudo apt-get -y install python3
+RUN sudo apt-get -y install python3-pip
+RUN sudo apt-get -y install python-is-python3
+RUN sudo apt-get -y install libpq-dev 
+RUN sudo apt-get -y install ruby-full rubygems
+RUN gem install sass
```

Every layer you add is an extra layer to the Docker image.

You want to string together as much as you can with `&&`.. i.e.

```suggestion
RUN apt-get -y update && apt-get -y clean && apt-get -y install sudo git python3 python3-pip python-is-python3 libpq-dev ruby-full rubygems && gem install sass
```